src/main/scala/prci/ClockNodes.scala

```diff
@@ -73,7 +73,7 @@ object ClockSourceNode
 object ClockGroupImp extends SimpleNodeImp[ClockGroupSourceParameters, ClockGroupSinkParameters, ClockGroupEdgeParameters, ClockGroupBundle]
 {
   def edge(pd: ClockGroupSourceParameters, pu: ClockGroupSinkParameters, p: Parameters, sourceInfo: SourceInfo) = ClockGroupEdgeParameters(pd, pu, p, sourceInfo)
-  def bundle(e: ClockGroupEdgeParameters) = new ClockGroupBundle(e.bundle)
+  def bundle(e: ClockGroupEdgeParameters) = new ClockGroupBundle(e.bundle, Some(e.bundle.members.zipWithIndex.map{ case (_, i) => s"${e.sink.name}_${i}"}))
```

Comment:
@hcook I am right that only the `sink` has a name, right? there is no `source` so I can't do something like "source_to_sink" right?

Reply:
Correct, though such functionality would be an argument for adding such a parameter. The source-side parameters are very underdeveloped. We could also think about adding the NodePath functionality that some of the other protocols have, then  lieu of a parameterized name you might be able to get the ValName of the source node or the module name of it's parent.

Reply:
But even just knowing the sink is useful IMO